bin/test-without-stubs

```diff
@@ -19,6 +19,9 @@ for dir in exercises/*/; do
   fi
 done
 
+# This raises many problems since it's an empty file
+mv exercises/concept/lasagna/lasagna_test.go exercises/concept/lasagna/lasagna_test.notgo
```

Comment:
can we just make it non-empty? It just needs a package name, then it should not create problems, right?

Reply:
apart from the fact that tests are needed...

Reply:
The problem within the lasagna (but also for every practice test) is the tests are calling methods not declared yet in the .go counterpart. These packages are impossible to build without adding empty methods for every exercise.

As per saying, the lasagna concepts, should teach to write methods, so this one cannot be fixed in any way. This will never build.

Reply:
ok, got it.